resources/js/feature_post.js

```diff
@@ -0,0 +1,188 @@
+  
+    card.classList.add('card');
+    card.appendChild(featureimage);
+    card.appendChild(content);
+  
+    cardcontainer.classList.add('card-container');
+    cardcontainer.appendChild(card);
+  
+    listcard.appendChild(cardcontainer);
+  }
+ const feature_post_2 = [
+    {
+      image: 'resources/images/abramovich.jpg',
+      title: '7. Super ultra hot videos Super ultra hot videos Super ultra hot videos',
+      view: ' 30.000',
+    },
+    {
+      image: 'resources/images/abramovich.jpg',
+      title: '8. Super ultra hot videos Super ultra hot videos Super ultra hot videos',
+      view: ' 20.000',
+    },
+    {
+      image: 'resources/images/abramovich.jpg',
+      title: '9. Super ultra hot videos Super ultra hot videos Super ultra hot videos',
+      view: ' 10.000',
+    },
+  ];
+  
+  var listcard = document.getElementById('feature-posts').getElementsByClassName('card-list')[2];
+    for (var i = 0; i < feature_post_2.length; i++) {
+      var icon = document.createElement('I');
+      var featureview = document.createElement('DIV');
+      var featuretitle = document.createElement('DIV');
+      var content = document.createElement('DIV');
+      var featureimage = document.createElement('IMG');
+      var card = document.createElement('DIV');
+      var cardcontainer = document.createElement('DIV');
+    
+      featureview.classList.add('feature-view');
+      icon.className = 'fas fa-eye';
+      featureview.appendChild(icon);
+      var fview = document.createTextNode(feature_post_2[i].view);
+      featureview.appendChild(fview);
+    
+      featuretitle.classList.add('feature-title');
+      var ftitle = document.createTextNode(feature_post_2[i].title);
+      featuretitle.appendChild(ftitle);
+    
+      content.classList.add('content');
+      content.appendChild(featuretitle);
+      content.appendChild(featureview);
+    
+      featureimage.classList.add('feature-image');
+      featureimage.setAttribute('src', feature_post_2[i].image);
+    
+      card.classList.add('card');
+      card.appendChild(featureimage);
+      card.appendChild(content);
+    
+      cardcontainer.classList.add('card-container');
+      cardcontainer.appendChild(card);
+    
+      listcard.appendChild(cardcontainer);
+    }
+  
+var slideIndex = 1;
+showSlides(slideIndex);
+// Thumbnail image controls
+function Slide(n) {
+  showSlides(slideIndex = n);
+}
+
+function showSlides(n) {
+  var i;
+  var slides = document.getElementsByClassName("card-list");
+  var dots = document.getElementsByClassName("dot");
+  if (n > slides.length) {slideIndex = 1}
+  if (n < 1) {slideIndex = slides.length}
+  for (i = 0; i < slides.length; i++) {
+      slides[i].style.display = "none";
+  }
+  for (i = 0; i < dots.length; i++) {
+      dots[i].className = dots[i].className.replace(" active", "");
+  }
+  slides[slideIndex-1].style.display = "flex";
+  dots[slideIndex-1].className += "active";
+}
+
```

Comment:
No newline
And if you have time, please refactor your JS implementation, it's too complicated for a slider
Check again your slider UI. It has bugs. Next time. please check thoroughly before making a Pull Request